packages/components/pie-notification/src/notification.scss

```diff
@@ -1 +1,91 @@
 @use '@justeattakeaway/pie-css/scss' as p;
+@use '@justeat/pie-design-tokens/dist/jet.scss' as dt;
```

Comment:
I can't see this namespace being used, why is this import needed?

Reply:
Removed